src/v/cluster/plugin_frontend.cc

```diff
@@ -534,6 +542,15 @@ errc plugin_frontend::validator::validate_mutation(const transform_cmd& cmd) {
                     out_name.tp);
                   return errc::transform_invalid_create;
               }
+              if (_topics->is_fully_disabled(out_name)) {
```

Comment:
nit: if not too much effort consider writing a test in `plugin_frontend_validator_test.cc`

Reply:
will follow up with this